src/Builder/Processor/ArchiveProcessor.php

```diff
@@ -61,7 +61,7 @@ protected function generateArchivePage(int $year, array $meetups): void
 		}
 
 		$meetups = array_map(fn(MeetupEntry $meetupEntry) => $meetupEntry->instance, $meetups);
-		rsort($meetups);
+		usort($meetups, fn($a, $b) => $b->getDateTime() <=> $a->getDateTime());
```

Comment:
There's already a sort method on the (doubly-linked list) collection class:
https://github.com/MergePHP/mergephp-website/blob/d0f2836a317101136f931b9ac7dbd04a61df5035/src/Builder/MeetupCollection.php#L18

That method is already called on the instance that is injected into this class on line 18.  So if the collection is already sorted by the time it arrives at this line I'm not sure why rsort doesn't work.  A cursory glance has left me more confused than when I started.

Once we get to the bottom of that it'd be nice to have a test around this so we can't break it in the next refactor

Reply:
There $meetups is just an array of MeetupEntry rather than the collection.
I think rsort doesn't know we want MeetupEntries sorted by date.

Also, for what it's worth, if I implement the sort on the collection it messes up the order of the RSS feed too which I really don't want to mess with.